portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -103,7 +102,20 @@ function populateDropdown(list, ID) {
   });
 }
 
+// inserts a functioning searchbar into the navigation bar of a page.
 function insertSearch() {
+  const searchVisual = createSearch();
```

Comment:
@kennzoid comment:
https://github.com/kofimeighan/adelardos-capstone-2020/pull/17#discussion_r450630709
nit: Maybe call this searchElement and createSearchElement?